c/src/ml-api-service-offloading.h

```diff
@@ -14,17 +14,61 @@
 #define __ML_SERVICE_OFFLOADING_INTERNAL_H__
 
 #include <ml-api-service.h>
+#include <nnstreamer-edge.h>
+
 #include "nnstreamer-tizen-internal.h"
 
 #ifdef __cplusplus
 extern "C" {
 #endif /* __cplusplus */
 typedef void * ml_service_offloading_h;
 
+/**
+ * @brief Enumeration for ml-offloading type.
+ */
+typedef enum
+{
+  ML_OFFLOADING_TYPE_UNKNOWN = 0,
+  ML_OFFLOADING_TYPE_TRAINING,
```

Comment:
Uh.. what about the traditional inference offloading?

Reply:
Yes, I will consider it. next PR.